.github/workflows/ci.yml

```diff
@@ -95,7 +95,11 @@ jobs:
         java-version: 8
 
     - name: Build and publish to Bintray/MavenCentral
-      run: ./gradlew --debug bintrayUpload githubRelease
+      run: # Print output every minute to avoid Github Actions timeout
+           while sleep 1m; do echo "=====[ $SECONDS seconds elapsed -- still running ]====="; done &
+           ./gradlew bintrayUpload githubRelease
+           # Killing background sleep loop
+           kill %1
```

Comment:
Probably we should use multiline yaml syntax

```yml
 - run: |
  line1 
  line2
```
  to preserve line end sing, in other case we have one line with all command, so first line contains comments and nothing will be executed

Reply:
Where is set and updated `SECONDS` variable ?

Reply:
ok `SECONDS` - is from bash ;-)

Reply:
You're right, the code in workflow file couldn't be executed written like it was earlier. I used yaml multiline syntax as you requested to correct it.